spdx-utils/src/main/kotlin/Extensions.kt

```diff
@@ -52,7 +52,7 @@ infix fun SpdxLicense.or(other: SpdxExpression) =
  * Create an [SpdxExpression] by concatenating [this][SpdxLicense] and [exception] using [SpdxOperator.WITH].
```

Comment:
`SpdxOperator.WITH` does not exist anymore and thus should not be referred in KDoc.

Reply:
Done.